src/server/scripts/Spells/spell_warlock.cpp

```diff
@@ -967,7 +967,7 @@ class spell_warl_life_tap : public SpellScriptLoader
 
         SpellCastResult CheckCast()
         {
-            if ((int32(GetCaster()->GetHealth()) > int32(GetSpellInfo()->Effects[EFFECT_0].CalcValue() + (6.3875 * GetSpellInfo()->BaseLevel) + (GetCaster()->GetStat(STAT_SPIRIT) * 1.5f) + 1.0f )))
+            if ((int32(GetCaster()->GetHealth()) > int32(GetSpellInfo()->Effects[EFFECT_0].CalcValue() + (3.1 * GetSpellInfo()->BaseLevel) + 0.9f )))
```

Comment:
shall we move these magic numbers inside constants? especially the `0.9f` as it's repeated

Reply:
`damage int32 = GetEffectValue () + 0.9f;`

**Specifies the amount of life subtracted.**

Reply:
I meant the `0.9f`